index.html

```diff
@@ -37,6 +32,119 @@ <h3>I’am  web a developer.</h3>
             <li><img src="images/medium.png" alt="social-icon" /></li>
       </ul>   
     </section>
+       <section class="my-works1">
+
+
+        <h1>My Recent works</h1>
+
+     
+        <h3>Multi-Post Stories</h3>
+        <a class="img1"><img src="images/placeholder.png" alt="hamburger Icon" /></a>
```

Comment:
- [ ] [Optional] This `a` tag is not properly used here, `a` tags create links to internal or external sources, in this case having the `img` directly in the HTML document will do the job